### PR TITLE
Add flag for using service user or not

### PR DESCRIPTION
**Pull request by re-ereit**

### CODE REVIEW

File: rexos/client.go [hunk 1]
```diff
@@ -51,21 +51,25 @@ type JwtToken struct {
...
 // NewClient create a new REXos HTTP client
-func NewClient(cfg Config) *Client {
-
+func NewClient(cfg Config, useServiceUser bool) *Client {
 	client := &Client{
-		httpClient: http.DefaultClient,
-		config:     cfg,
+		httpClient:     http.DefaultClient,
+		config:         cfg,
+		useServiceUser: true,
```
Comment (breiting) [inline rexos/client.go:66]: useServiceUser: useServiceUser

File: rexos/client.go [hunk 2]
```diff
@@ -51,21 +51,25 @@ type JwtToken struct {
...
 	}
+	client.useServiceUser = useServiceUser
```
Comment (breiting) [inline rexos/client.go:68]: remove this line

File: rexos/service.go [hunk 3]
```diff
@@ -23,10 +23,13 @@ type Service struct {
 }
 
 // NewService returns a new rexos service which is implementing the RexOSAccessor interface
-func NewService(config Config) *Service {
-
+func NewService(config Config, args ...bool) *Service {
+	useServiceUser := true
+	if len(args) > 0 {
```
Comment (breiting) [inline rexos/service.go:28]: this is dangerous .. why do not put it into the config?

File: rexos/client.go (hunk 1 above)
Comment (re-ereit) [reply rexos/client.go:66]: done

File: rexos/client.go (hunk 2 above)
Comment (re-ereit) [reply rexos/client.go:68]: removed

**Pull request merged**